### PR TITLE
feat(scheduler): generate template

### PR DESCRIPTION
**Pull request by InteractiveTimmy**
# Description

The scope of the changes in this pull request are to create a template plugin for the team responsible for implementing the `scheduler` featureset.

## Links

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-403126

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -285,6 +285,10 @@ module.exports = {
     'webex-core.js',
     'ediscovery.js',
     'CHANGELOG.md',
+    'README.md',
+    'LICENSE.md',
+    'package.json',
+    'yarn.lock',
```
Comment (InteractiveTimmy) [inline .eslintrc.js:291]: Was getting interesting ESLint issue over this, in which it was unable to validate these files.

If anyone knows a better solution, I'm welcome to making the changes necessary.

**Comment by chrisadubois** [comment]
@InteractiveTimmy - this is great, thank you for doing this - we just need some documentation around it.

File: packages/@webex/internal-plugin-scheduler/src/scheduler/scheduler.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      /* ...perform actions once the webex object is fully qualified... */
+    });
+  }
+
+  /**
+   * Example request usage. `this.request` returns a promise using XHR. and the
+   * `request` npm package.
+   * @returns {void}
+   */
+  exampleRequestUsage() {
```
Comment (chrisadubois) [inline packages/@webex/internal-plugin-scheduler/src/scheduler/scheduler.js:65]: can you move this stuff into some sub folders as examples as well? it doesn't all need to be done in the scheduler.js file

Comment (InteractiveTimmy) [reply packages/@webex/internal-plugin-scheduler/src/scheduler/scheduler.js:65]: I considered this, and spent some time writing some templates like this. But, decided ultimately against doing this, as the architect/director of this plugin should have the freedom to adjust folder structure, relations between models, and event pipelines as needed.

I will stand by for QA for best practices and general inquiries from the team owning this plugin.

**Review by arun3528 — APPROVED** [review]

**Review by chrisadubois — APPROVED** [review]

**Pull request merged**